site/frontend/src/pages/compare/compile/table/shortcuts/profile-shortcut.vue

```diff
@@ -0,0 +1,82 @@
+<script setup lang="ts">
+import {computed, ref, Ref} from "vue";
```

Comment:
If we're documenting the `CachegrindCmd` component above, then let's also do it for this shortcut file as well as the binary-size shortcut file.